crypto/rand/sysrand.c

```diff
@@ -58,11 +57,15 @@ int GFp_sysrand_chunk(void *buf, size_t len);
 
 #pragma warning(pop)
 
-const size_t GFp_sysrand_chunk_max_len = ULONG_MAX;
-
 int GFp_sysrand_chunk(void *out, size_t requested) {
-  assert(requested <= GFp_sysrand_chunk_max_len);
-  return RtlGenRandom(out, (ULONG)requested) ? 1 : 0;
+  if (requested > ULONG_MAX) {
+    requested = ULONG_MAX;
+  }
+  if (requested > (size_t) INT_MAX) {
```

Comment:
Nit: No space between the cast and the value, according to the BoringSSL style guide.